aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,193 @@
+
+This article includes a [sample app](#download10) that demonstrates the [`Microsoft.AspNetCore.OpenApi`](https://www.nuget.org/packages/Microsoft.AspNetCore.OpenApi) package's ability to integrate XML documentation comments on types into OpenAPI documents. The sample app is a minimal ASP.NET Core Web API project that uses the `Microsoft.AspNetCore.OpenApi` package to generate OpenAPI documents. The XML documentation comments are used to populate summaries, descriptions, parameter information, and response details in the generated OpenAPI document.
+
+![screenshot of app with XML comments in sclaar UI](~/fundamentals/openapi/_static/screenshot.png)
+
+## Customizing XML documentation behavior
+
+The following sections describe how to enable and customize XML documentation support.
+
+### Enable XML documentation in an ASP.NET Core API project
+
+1. Enable XML documentation in the project file:
+
+    :::code language="xml" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/models/Models.csproj" highlight="7":::
+
+2. Use the [AddOpenApi](xref:Microsoft.Extensions.DependencyInjection.OpenApiServiceCollectionExtensions.AddOpenApi*) method in the service configuration. No configuration is needed, the source generator handles the rest.
+
+    :::code language="csharp" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/api/Program.cs" highlight="6":::
+
+The source generator detects all standard overloads:
+
+* [`AddOpenApi()`](xref:Microsoft.Extensions.DependencyInjection.OpenApiServiceCollectionExtensions.AddOpenApi(Microsoft.Extensions.DependencyInjection.IServiceCollection))
+* [`AddOpenApi("v1")`](xref:Microsoft.Extensions.DependencyInjection.OpenApiServiceCollectionExtensions.AddOpenApi(Microsoft.Extensions.DependencyInjection.IServiceCollection,System.String))
+* [`AddOpenApi(options => {})`](xref:Microsoft.Extensions.DependencyInjection.OpenApiServiceCollectionExtensions.AddOpenApi(Microsoft.Extensions.DependencyInjection.IServiceCollection,System.Action{Microsoft.AspNetCore.OpenApi.OpenApiOptions}))
+* [`AddOpenApi("v1", options => {})`](xref:Microsoft.Extensions.DependencyInjection.OpenApiServiceCollectionExtensions.AddOpenApi(Microsoft.Extensions.DependencyInjection.IServiceCollection,System.String,System.Action{Microsoft.AspNetCore.OpenApi.OpenApiOptions}))
+
+Each overload is intercepted to automatically include the XML documentation transformers. The source generator doesn't handle overloads where the `documentName` parameter isn't a literal string expression. For example, the transformer isn't registered in the following scenarios:
+
+```csharp
+var documentName = "v1";
+builder.Services.AddOpenApi(documentName); // No XML support
+```
+
+### Adding XML documentation sources
+
+The [`Microsoft.AspNetCore.OpenApi`](https://www.nuget.org/packages/Microsoft.AspNetCore.OpenApi) NuGet package automatically resolves XML documentation comments from:
```

Comment:
We refer to the thing that performs XML documentation comment processing in several different ways in this article, and I think we could avoid confusion if we settled on just one way. My preference would be "ASP.NET Core", as used in the third paragraph (that I suggested be moved to the second paragraph), rather than "OpenAPI XML documentation processing" (in the first paragraph) or "The `Microsoft.AspNetCore.OpenApi` NuGet package`", as in this paragraph.

Reply:
Can you take a quick look at my changing from "OpenAPI XML documentation processing"?
I made this change except to the title because it seems like it needs OpenAPI XML in the title. Please suggest better wording.